cmd/kubelet/app/server.go

```diff
@@ -476,7 +477,9 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, featureGate f
 	if err := options.ValidateKubeletServer(s); err != nil {
 		return err
 	}
-
+	if len(s.LogFormat) > 0 {
+		logs.NewOptions().Apply()
```

Comment:
Do we need to use the LogFormat option here?

Reply:
We could probably initialize this at the beginning of `Run` instead of `run`, or even right at the end of `cmd.Run`, just before we log the config and call `Run`, so we're a little earlier in the startup flow.

Reply:
I add it to the beginning of `Run`.